debian/install.sh

```diff
@@ -361,42 +489,70 @@ timewait 2
 PMODWAIT=0
 
 # remediate
-if [ $CPANOPTION == 1 ]; then
-for i in "${ARMOD[@]}"
-do
-	perldoc -l $i >/dev/null 2>&1
+if [ ${CPANOPTION} == 1 ]; then
+    #Install pre SpamAssassin modules
+    for i in "${ARMOD[@]}"
+    do
+        perldoc -l ${i} >/dev/null 2>&1
+	    if [ $? != 0 ]; then
+            clear
+            echo "${i} is missing. Installing via CPAN ..."; echo;
+            timewait 1
+            perl -MCPAN -e "CPAN::Shell->force(qw(install ${i} ));"
+        fi
+    done
+
+    #Install SpamaAssassin, use standard cpan in normail install, or App::cpanminus in unattended install
```

Comment:
Small typo `normail` and I'm not sure if all variables nees to be extended by using `${}`